src/messages/hu/admin.php

```diff
@@ -0,0 +1,461 @@
+<?php
+
+return [
+    'language_name' => 'angol',
```

Comment:
@mrbig00 This should be name name of the language hungarian, is it `Magyar`?

Reply:
my bad

I updated the file